lib/geokit-rails/acts_as_mappable.rb

```diff
@@ -93,7 +93,7 @@ module ClassMethods
       # A proxy to an instance of a finder adapter, inferred from the connection's adapter.
       def adapter
         @adapter ||= begin
-          require File.join('geokit-rails', 'adapters', connection.adapter_name.downcase)
+          require File.join(File.dirname(__FILE__), 'adapters', connection.adapter_name.downcase)
```

Comment:
Line is too long. [97/80]<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.